src/Entity/Adherent.php

```diff
@@ -1198,8 +1198,18 @@ public function setChartAccepted(bool $chartAccepted): void
         $this->chartAccepted = $chartAccepted;
     }
 
-    public function getMandate(): ?string
+    public function getMandates(): ?array
+    {
+        return $this->mandates;
+    }
+
+    public function setMandates(array $mandates): void
+    {
+        $this->mandates = $mandates;
+    }
+
+    public function hasMandate(): bool
     {
-        return $this->mandate;
+        return \count($this->mandates) > 0;
```

Comment:
it'll trigger an error it `$this->mandates` is null

Reply:
and what about `isElectedPerson` as the name of method?

Reply:
You can just use:
```php
return !empty($this->mandates)
```

Reply:
:heavy_check_mark: